### ISSUE TITLE
Failed to instantiate component or class "verbb\base\assetbundles\CpAsset".

### ISSUE THREAD

**Issue by sandrodunkel** [vandres/craft-matrix-extended#10]
Just tried to install Matrix Entended on a project i just recently upgraded from Craft 4 to Craft 5 and i get the following error:

### Stack Trace
```
ReflectionException: Class "verbb\base\assetbundles\CpAsset" does not exist in /Users/username/dev/projectname/vendor/yiisoft/yii2/di/Container.php:507
Stack trace:
#0 /Users/username/dev/projectname/vendor/yiisoft/yii2/di/Container.php(507): ReflectionClass->__construct('verbb\\base\\asse...')
#1 /Users/username/dev/projectname/vendor/yiisoft/yii2/di/Container.php(385): yii\di\Container->getDependencies('verbb\\base\\asse...')
#2 /Users/username/dev/projectname/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build('verbb\\base\\asse...', Array, Array)
#3 /Users/username/dev/projectname/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('verbb\\base\\asse...', Array, Array)
#4 /Users/username/dev/projectname/vendor/yiisoft/yii2/web/AssetManager.php(292): yii\BaseYii::createObject(Array)
#5 /Users/username/dev/projectname/vendor/yiisoft/yii2/web/AssetManager.php(265): yii\web\AssetManager->loadBundle('verbb\\base\\asse...', Array, true)
#6 /Users/username/dev/projectname/vendor/yiisoft/yii2/web/View.php(296): yii\web\AssetManager->getBundle('verbb\\base\\asse...')
#7 /Users/username/dev/projectname/vendor/craftcms/cms/src/web/View.php(2106): yii\web\View->registerAssetBundle('verbb\\base\\asse...', NULL)
#8 /Users/username/dev/projectname/vendor/yiisoft/yii2/web/View.php(301): craft\web\View->registerAssetBundle('verbb\\base\\asse...', NULL)
#9 /Users/username/dev/projectname/vendor/craftcms/cms/src/web/View.php(2106): yii\web\View->registerAssetBundle('vandres\\matrixe...', NULL)
#10 /Users/username/dev/projectname/vendor/vandres/craft-matrix-extended/src/MatrixExtended.php(87): craft\web\View->registerAssetBundle('vandres\\matrixe...')
#11 /Users/username/dev/projectname/vendor/craftcms/cms/src/base/ApplicationTrait.php(489): vandres\matrixextended\MatrixExtended->vandres\matrixextended\{closure}()
#12 [internal function]: craft\web\Application->craft\base\{closure}(Object(yii\base\Event))
#13 /Users/username/dev/projectname/vendor/yiisoft/yii2/base/Component.php(633): call_user_func(Object(Closure), Object(yii\base\Event))
#14 /Users/username/dev/projectname/vendor/craftcms/cms/src/base/ApplicationTrait.php(530): yii\base\Component->trigger('init', Object(yii\base\Event))
#15 /Users/username/dev/projectname/vendor/craftcms/cms/src/base/ApplicationTrait.php(1646): craft\web\Application->trigger('init')
#16 /Users/username/dev/projectname/vendor/craftcms/cms/src/web/Application.php(110): craft\web\Application->_postInit()
#17 /Users/username/dev/projectname/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\web\Application->init()
#18 /Users/username/dev/projectname/vendor/yiisoft/yii2/base/Application.php(204): yii\base\BaseObject->__construct(Array)
#19 [internal function]: yii\base\Application->__construct(Array)
#20 /Users/username/dev/projectname/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs(Array)
#21 /Users/username/dev/projectname/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build('craft\\web\\Appli...', Array, Array)
#22 /Users/username/dev/projectname/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('craft\\web\\Appli...', Array, Array)
#23 /Users/username/dev/projectname/vendor/craftcms/cms/src/Craft.php(70): yii\BaseYii::createObject(Array, Array)
#24 /Users/username/dev/projectname/vendor/craftcms/cms/bootstrap/bootstrap.php(316): Craft::createObject(Array)
#25 /Users/username/dev/projectname/vendor/craftcms/cms/bootstrap/web.php(40): require('/Users/usern...')
#26 /Users/username/dev/projectname/web/index.php(25): require('/Users/usern...')
#27 {main}

Next yii\di\NotInstantiableException: Failed to instantiate component or class "verbb\base\assetbundles\CpAsset". in /Users/username/dev/projectname/vendor/yiisoft/yii2/di/Container.php:509
Stack trace:
#0 /Users/username/dev/projectname/vendor/yiisoft/yii2/di/Container.php(385): yii\di\Container->getDependencies('verbb\\base\\asse...')
#1 /Users/username/dev/projectname/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build('verbb\\base\\asse...', Array, Array)
#2 /Users/username/dev/projectname/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('verbb\\base\\asse...', Array, Array)
#3 /Users/username/dev/projectname/vendor/yiisoft/yii2/web/AssetManager.php(292): yii\BaseYii::createObject(Array)
#4 /Users/username/dev/projectname/vendor/yiisoft/yii2/web/AssetManager.php(265): yii\web\AssetManager->loadBundle('verbb\\base\\asse...', Array, true)
#5 /Users/username/dev/projectname/vendor/yiisoft/yii2/web/View.php(296): yii\web\AssetManager->getBundle('verbb\\base\\asse...')
#6 /Users/username/dev/projectname/vendor/craftcms/cms/src/web/View.php(2106): yii\web\View->registerAssetBundle('verbb\\base\\asse...', NULL)
#7 /Users/username/dev/projectname/vendor/yiisoft/yii2/web/View.php(301): craft\web\View->registerAssetBundle('verbb\\base\\asse...', NULL)
#8 /Users/username/dev/projectname/vendor/craftcms/cms/src/web/View.php(2106): yii\web\View->registerAssetBundle('vandres\\matrixe...', NULL)
#9 /Users/username/dev/projectname/vendor/vandres/craft-matrix-extended/src/MatrixExtended.php(87): craft\web\View->registerAssetBundle('vandres\\matrixe...')
#10 /Users/username/dev/projectname/vendor/craftcms/cms/src/base/ApplicationTrait.php(489): vandres\matrixextended\MatrixExtended->vandres\matrixextended\{closure}()
#11 [internal function]: craft\web\Application->craft\base\{closure}(Object(yii\base\Event))
#12 /Users/username/dev/projectname/vendor/yiisoft/yii2/base/Component.php(633): call_user_func(Object(Closure), Object(yii\base\Event))
#13 /Users/username/dev/projectname/vendor/craftcms/cms/src/base/ApplicationTrait.php(530): yii\base\Component->trigger('init', Object(yii\base\Event))
#14 /Users/username/dev/projectname/vendor/craftcms/cms/src/base/ApplicationTrait.php(1646): craft\web\Application->trigger('init')
#15 /Users/username/dev/projectname/vendor/craftcms/cms/src/web/Application.php(110): craft\web\Application->_postInit()
#16 /Users/username/dev/projectname/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\web\Application->init()
#17 /Users/username/dev/projectname/vendor/yiisoft/yii2/base/Application.php(204): yii\base\BaseObject->__construct(Array)
#18 [internal function]: yii\base\Application->__construct(Array)
#19 /Users/username/dev/projectname/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs(Array)
#20 /Users/username/dev/projectname/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build('craft\\web\\Appli...', Array, Array)
#21 /Users/username/dev/projectname/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('craft\\web\\Appli...', Array, Array)
#22 /Users/username/dev/projectname/vendor/craftcms/cms/src/Craft.php(70): yii\BaseYii::createObject(Array, Array)
#23 /Users/username/dev/projectname/vendor/craftcms/cms/bootstrap/bootstrap.php(316): Craft::createObject(Array)
#24 /Users/username/dev/projectname/vendor/craftcms/cms/bootstrap/web.php(40): require('/Users/usern...')
#25 /Users/username/dev/projectname/web/index.php(25): require('/Users/usern...')
#26 {main}
```

### Craft and plugin versions:
- Craft Pro 5.1.8
- PHP Version 8.2.0
- CKEditor 4.0.6
- Cookies 5.0.0
- Dashboard Begone 3.0.0
- SEOmatic 5.0.3

**Comment by vandres** [comment]
Thanks for reporting! Will release a fix asap.

**Comment by vandres** [comment]
@sandrodunkel Should be fixed in 3.3.1
